src/igvfd/schemas/auxiliary_set.json

```diff
@@ -61,6 +61,7 @@
                 "circularized RNA barcode detection",
                 "gRNA sequencing",
                 "lipid-conjugated oligo sequencing",
+                "MORF barcode sequencing",
```

Comment:
This is good, I would just add a sentence in the above "comment" (line 57) to explain the new term. https://www.addgene.org/pooled-library/human-morf-library/ and https://pubmed.ncbi.nlm.nih.gov/38697106/ are the relevant info. Just want to especially include it since the "ORF" in this is not the usual "open reading frame" so it may be more confusing than average.

I propose something like:
"MORF barcode sequencing is the sequencing of the Multiplexed Overexpression of Regulatory Factors library to produce a mapping of barcodes to an overexpressed transcription factor."